configure.ml

```diff
@@ -198,8 +198,8 @@ type nativecompiler = NativeYes | NativeNo | NativeOndemand
 
 type preferences = {
   prefix : string option;
-  local : bool;
   interactive : bool;
+  output_summary : bool;
```

Comment:
What's this new flag for?

Reply:
Before we used interactive to both output configure summary and stop asking for `-prefix` interactively.

Reply:
Now these are separate cases, but indeed we should rework all of the code there.